### PR TITLE
Qa screen

### PR DESCRIPTION
**Pull request by samiur98**
- Implemented the QAScreen that displays right answers, wrong answers, and/or ungraded answers after a task is completed

### CODE REVIEW

File: src/Components/ProgressBar.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default function ProgressBar(props) {
     return(
         <div className='pbar' style = {barStyle}>
             { getDone() }
-            { getLeft() }
+            { getRemaining() }
```
Comment (Bart-Ilg) [inline src/Components/ProgressBar.js:64]: good renames

File: src/Components/QAScreen.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.scroll-grid-wrapper {
+    display: 'flex';
+    flex-wrap: 'wrap';
+    justify-content: 'space-around';
+    overflow: 'hidden';
+}
```
Comment (Bart-Ilg) [inline src/Components/QAScreen.css:6]: looks clean

File: src/Components/QAScreen.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { GridList, GridListTile } from '@material-ui/core';
+import './QAScreen.css';
+
+// Sample Data.
+
+const sampleData = [
```
Comment (Bart-Ilg) [inline src/Components/QAScreen.js:7]: good sample for now, will change to props

File: src/Components/QAScreen.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    }
+    if(pointsAwarded/points >= 1) {
+      return '#27AE60';
+    }
+    return '#EB5757';
+  }
+
+  function ScrollableGridList() {
+    return(
+      <GridList cellHeight={spacing} className='scroll-grid' cols={1} style={{'height' : `${height}`}}>
```
Comment (Bart-Ilg) [inline src/Components/QAScreen.js:81]: Well Organized, look forward to integrating it

**Review by Bart-Ilg — APPROVED** [review]
Component looks clear, I look forward to merging it with mine so we can add it to the project

**Review by ConnorBoulais — APPROVED** [review]
Looks good

**Pull request merged**